src/audio/kpb.c

```diff
@@ -731,11 +727,9 @@ static int kpb_buffer_data(struct comp_dev *dev, struct comp_buffer *source,
 		/* Are we stuck in buffering? */
 		current_time = platform_timer_get(platform_timer);
 		if (timeout < current_time) {
-			trace_kpb_error_with_ids(dev, "kpb_buffer_data(): "
-					"timeout of %d [ms] "
-					"(current state %d, state log %d)",
-					(current_time - timeout),
-					kpb->state, kpb->state_log);
+			trace_kpb_error("kpb_buffer_data(): timeout of %d [ms] (current state %d, state log %x)",
+					(current_time - timeout), kpb->state,
```

Comment:
these parentheses are superfluous, since you're touching this code, you could as well remove them